packages/ui-react/src/components/TileDock/TileDock.tsx

```diff
@@ -203,6 +204,10 @@ function TileDock<T>({
         setSlideToIndex(resetIndex);
       }
 
+      if (firstVisibleTileIndex !== -1) {
+        (frameRef.current?.children[firstVisibleTileIndex] as HTMLElement).focus();
```

Comment:
This will cause the screen reader to read the slide item content on the left. Based on the browser width this could be one item only. So you will toggle between a slide item and chevron button on every (screen reader navigation) swipe

Reply:
When you use optional chaining, you always want to prevent casting it as non-nullish. I think we can improve this behavior, but I currently don't have a better suggestion.

```suggestion
        (frameRef.current?.children[firstVisibleTileIndex] as HTMLElement | undefined)?.focus();
```

Reply:
I'm not sure if we want to focus the card programmatically after sliding. This will also cause the first visible card to be focussed when pressing the slide button with the mouse.

Reply:
Yes, and (about desktop) we agreed that the slider is currently more accessible as it is, than moving the focus from script. 
Can we not just skip this part? I think the screen reader user knows what to expect and will move the screen reader selector back to the tile.

Reply:
> When you use optional chaining, you always want to prevent casting it as non-nullish. 

Though I find it a bit weird to use an optional chaining before .`focus()` and adding `| undefined` because we can assume children only having elements of type `HTMLElement`. While also making sure the index exists beforehand. 

Maybe something like this is more approriate (but still not complete): `const elements: HTMLElement[] = Array.from(frameRef.current?.children || []);` and then using `elements[firstVisibleTileIndex].focus();`

> This will also cause the first visible card to be focussed when pressing the slide button with the mouse.

I have seen blueish focus outline once. But I cannot reproduce this anymore. When I press the chevron button on both chrome and safari on macOS I do not see any visual indication that the first visible card is focussed. 

For the browser which DOES show a browser native outline. Maybe this is something we need to accept? Comparable are the yellow outline on the checkboxes on the DebatDirect search. Also not that pretty, but acceptable

I'm just throwing this up to think about. I believe we can leave it as this and make it better in the future?